### PR TITLE
Added some vagrant notes

### PR DESCRIPTION
**Pull request by jjasghar**
This is a quick page to have how to add forwarded ports and
another nic to your base .kitchen.yml.

I know I've searched a few times for it; it would be nice to
have in the docs.

### CODE REVIEW

File: source/docs/getting-started/drivers-configuration.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Vagrant Driver
+
+Sometimes you'd like to add another NIC to your vm. You might have a `forwarded_port` you'd like
+to add also. The following snippet is how.
+
+~~~yaml
+  name: vagrant
+  network:
+  - ["forwarded_port", {guest: 80, host: 8080}]
```
Comment (tfitch) [inline source/docs/getting-started/drivers-configuration.md]: Could you include the auto_correct attribute here, so an example of that is shown?  It's a darn shame when a node converge fails and that's the reason. 

`- ["forwarded_port", {guest: 80, host: 8080, auto_correct: true}]`

Thanks

Comment (jjasghar) [inline source/docs/getting-started/drivers-configuration.md]: Great idea. I'll update it now.

**Comment by tyler-ball** [comment]
Hey @jjasghar - we like the idea of this documentation, but don't want to add it into the current documentation layout.  The `Guide` is meant to be a narrative that walks people through their first time using Test Kitchen.  Information like this is reference material, and ends up looking like 'Appendix A', 'Appendix B', etc.

So I tagged this into the `Reference Documentation` milestone.  When we start designing the docs for more information like this, we will pull this in.  Thanks for the PR!

**Comment by cheeseplus** [comment]
Closing this one for the reasons listed above but would love to have the content re-submitted as an Appendix as suggested.

**Pull request closed without merging**